book/src/kubernetes-changelog.md

```diff
@@ -21,6 +21,8 @@ details on individual features, visit the [Features section](features.md).
 * Beta
     * [ReadWriteOncePod Access mode](https://kubernetes.io/blog/2023/04/20/read-write-once-pod-access-mode-beta/)
     * [Speed up SELinux volume relabeling using mounts](https://kubernetes.io/blog/2023/04/18/kubernetes-1-27-efficient-selinux-relabeling-beta/)
+    * [CSI node expand secret](https://github.com/kubernetes/enhancements/tree/master/keps/sig-storage/3107-csi-nodeexpandsecret)
+    * [Prevent unauthorized volume mode converstion](https://github.com/kubernetes/enhancements/issues/3141)
```

Comment:
Can you link to the KEP https://github.com/kubernetes/enhancements/tree/master/keps/sig-storage/3141-prevent-volume-mode-conversion to be consistent with other features?